config/database.yml

```diff
@@ -20,6 +20,9 @@ default: &default
   # For details on connection pooling, see rails configuration guide
   # http://guides.rubyonrails.org/configuring.html#database-pooling
   pool: 5
+  username: postgres
+  password: password
+  host: localhost
```

Comment:
It strikes me as wrong somehow to have the db password committed to the repo. Perhaps we should add it to the gitignore?